components/Cards/README.md

```diff
@@ -269,18 +269,19 @@ recommendations:
 
 Since assistive technologies visit all cards in a collection in a sequential order, it is often 
 easier to distinguish between elements that belong to different cards by aggregating all the 
-card's information to a single sentence.  This can be done by setting an appropriate 
+card's information so the card is read as a single sentence.  
+This can be done by setting an appropriate 
 [`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) 
 for the card. Additionally, set the card's 
-[`isAccessiblityElement`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
-to on. Cards are a container element and setting isAccessibiltElement for a container turns 
-off individually selecting its subelements.
+[`isAccessibilityElement`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
+to true. Cards are a container element and setting isAccessibiltyElement for a container 
+turns off individually selecting its subelements.
 
 Swift
 ```swift
   card.isAccessibilityElement = true
   card.accessibilityLabel = "Location (userLocation.name) is popular with users " +
-    "who enjoy (userLocation.popularActivityMatchingUserProfile(userProfile)."
+    "who enjoy (userLocation.popularActivityMatchingUserProfile(userProfile))"
```

Comment:
I thought value escapes needed to be preceded with a backslash?

Reply:
The backslash is there in the original *.md file. 
It looks like it's being removed during the readme generation process.

Reply:
Maybe we need an extra backslash in the source file?